components/02-elements/rating/_rating.scss

```diff
@@ -1,78 +1,103 @@
-$rating-font-size: $font-size-large;
-$rating-active   : $orange;
-
+$rating__size: 96px;
+$rating__item-size: $rating__size / 5;
 .rating {
-    unicode-bidi: bidi-override;
-    direction: rtl;
-    font-size: $rating-font-size;
+    width: $rating__size;
     cursor: pointer;
-    text-align: left;
 
-    span {
-        display: inline-block;
+    &--rate {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        direction: rtl;
+    }
+    &__rate-item {
         position: relative;
-
+        flex: 1 0 $rating__item-size;
+        overflow: hidden;
         &:hover {
-            &:before {
-                content: "\2605";
-                position: absolute;
-                color: $rating-active;
-            }
-
-            ~ span {
-                &:before {
-                    content: "\2605";
-                    position: absolute;
-                    color: $rating-active;
-                }
+            ~ .rating__rate-item span:before {
+                display: block;
             }
         }
     }
-
-    &__summary {
+    &__star {
         position: relative;
-        display: inline-block;
-        width: 92px;
-        letter-spacing: 1px;
-        height: $rating-font-size;
-        font-size: $rating-font-size;
-        line-height: $rating-font-size;
-        vertical-align: middle;
-
-        &:before {
-            content: '\2605' '\2605' '\2605' '\2605' '\2605';
+        height: $rating__item-size;
+        width: $rating__size;
+        &--rate {
+            width: $rating__item-size;
+        }
+        &:before { // inactive
             position: absolute;
+            bottom: 0;
             left: 0;
-            width: 100%;
-            height: 100%;
+            top: 0;
+            right: 0;
             display: block;
-            z-index: 1;
-            color: $gray;
+            width: $rating__size;
+            content: "";
+            height: $rating__item-size;
+            background-image: url("raw/components/01-globals/icons/files/star-border.svg");
```

Comment:
Icon not found in preview... IMO you have to add separate icon to `public/images`but I'm wondering if we can use here icon inline? I recommend build short sprite with icons used as css background